core/arch/arm/plat-imx/conf.mk

```diff
@@ -73,6 +74,11 @@ CFG_NS_ENTRY_ADDR ?= 0x80800000
 $(call force,CFG_TEE_CORE_NB_CORE,2)
 endif
 
+ifneq (,$(filter $(PLATFORM_FLAVOR),mx7dclsom))
+CFG_DDR_SIZE ?= 0x40000000
+CFG_UART_BASE ?= UART1_BASE
```

Comment:
Is CFG_NS_ENTRY_ADDR needed?

Reply:
It's not. If `CFG_NS_ENTRY_ADDR` is not defined, it will jump to the address that was passed in `lr` when OPTEE was invoked. If someone wants to specify `CFG_NS_ENTRY_ADDR` at build time, they can always do so from the command line.